lsp/client/jetbrains/gradle.properties

```diff
@@ -4,11 +4,17 @@ pluginGroup = com.github.mercari.grpcfederation
 pluginName = grpc-federation
 pluginRepositoryUrl = https://github.com/mercari/grpc-federation
 # SemVer format -> https://semver.org
-pluginVersion = 0.1.0
+pluginVersion = 0.2.0
 
 # Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 pluginSinceBuild = 233
-pluginUntilBuild = 241.*
+# pluginUntilBuild is intentionally not set to allow the plugin to be compatible with future IDE versions.
+# This is relatively safe because:
+# 1. This plugin uses stable LSP (Language Server Protocol) APIs which are unlikely to break
+# 2. The plugin has minimal IDE integration surface area
+# 3. Users can install and test on newer versions without waiting for plugin updates
+# If compatibility issues arise in future versions, we can add the restriction then.
+pluginUntilBuild = 
```

Comment:
To avoid having to update and release this every time a new version of IntelliJ comes out, I’m going to disable this setting.